builder/library.go

```diff
@@ -67,7 +67,7 @@ func (l *Library) Load(target string) (path string, err error) {
 	defer os.RemoveAll(dir)
 
 	// Precalculate the flags to the compiler invocation.
-	args := append(l.cflags(), "-c", "-Oz", "-g", "-ffunction-sections", "-fdata-sections", "-Wno-macro-redefined", "--target="+target, "-fdebug-prefix-map="+dir+"="+remapDir)
+	args := append(l.cflags(), "-v","-c", "-Oz", "-g", "-ffunction-sections", "-fdata-sections", "-Wno-macro-redefined", "--target="+target, "-fdebug-prefix-map="+dir+"="+remapDir)
```

Comment:
This appears to be a leftover from some debugging, and should be removed.

Reply:
Done.